controllers/lesson.js

```diff
@@ -9,6 +10,18 @@ const createLesson = async (req, res) => {
   res.status(201).json(newLesson)
 }
 
+const getLessons = async (req, res) => {
+  const { id: author } = req.user
+  const { title, sort, skip, limit } = req.query
+
+  const match = getMatchOptions({ author, title })
+
+  const lessons = await lessonService.getLessons(match, sort, parseInt(skip), parseInt(limit))
```

Comment:
i think it would be better to combine it in the object instead of passing 4 parameters

Reply:
I don't think it's necessary